### PR TITLE
Flow light dynamic

### PR DESCRIPTION
**Pull request by fatonramadani**
https://user-images.githubusercontent.com/456655/179012270-13f7d69c-a0d3-438f-9ffa-09396f772b73.mov

### CODE REVIEW

**Comment by rubenfiszel** [comment]
Looks really cool. I would maybe replace "Code Editor" with "Raw Javascript Mode", wdyt ?

File: frontend/src/lib/components/SchemaForm.svelte [hunk 1]
```diff
@@ -27,7 +27,7 @@
 				return
 			}
 
-			if (isCodeInjection(arg.value)) {
+			if (isCodeInjection(String(arg.value))) {
```
Comment (rubenfiszel) [inline frontend/src/lib/components/SchemaForm.svelte]: do not think it's correct, you do not want to do rawValueString -> JSON.parse -> stringify
It's better to do instead
arg.rawValue ?? arg.value

Comment (rubenfiszel) [inline frontend/src/lib/components/SchemaForm.svelte]:
```suggestion
			if (isCodeInjection(String(arg.rawValue ?? arg.value))) {
```

File: frontend/src/lib/components/ModuleStep.svelte [hunk 2]
```diff
@@ -30,7 +32,7 @@
 	)
 </script>
 
-<li class="flex flex-row flex-shrink max-w-full mx-auto mt-16">
+<li class="flex flex-row flex-shrink max-w-full mx-auto mt-20">
```
Comment (rubenfiszel) [inline frontend/src/lib/components/ModuleStep.svelte]:
```suggestion
<li class="flex flex-row flex-shrink max-w-full mx-auto mt-16">
```

File: frontend/src/lib/components/SchemaForm.svelte [hunk 3]
```diff
@@ -119,7 +137,7 @@ previous_result.${key}`
 					type={schema.properties[argName].type}
 					required={schema.required.includes(argName)}
 					bind:pattern={schema.properties[argName].pattern}
-					bind:valid={inputCheck[argName]}
+					bind:valid={isValid}
```
Comment (rubenfiszel) [inline frontend/src/lib/components/SchemaForm.svelte]: ?
Seemed correct before

File: frontend/src/lib/components/SchemaForm.svelte (hunk 1 above)
Comment (rubenfiszel) [reply frontend/src/lib/components/SchemaForm.svelte]: nvm suggestion but that still seems wrong

File: frontend/src/lib/components/SchemaForm.svelte (hunk 3 above)
Comment (fatonramadani) [reply frontend/src/lib/components/SchemaForm.svelte]: Reverted

**Pull request merged**